app/routes/index.js

```diff
@@ -32,7 +32,6 @@ function regGlobalMiddleware(app) {
 module.exports = function init(app) {
   regGlobalMiddleware(app);
   app.get('/', statusRoute);
-  app.use('/v1/status', statusRoute);
```

Comment:
We have a Runscope test that uses this route. https://www.runscope.com/radar/e6s8uo1kkh7r/bcd0fa4a-d5c2-492f-8b41-49f10f176ba4

Reply:
Great catch! I've updated the test.